ivy/data_classes/array/array.py

```diff
@@ -401,13 +400,7 @@ def __repr__(self):
                 self._post_repr = ")"
         sig_fig = ivy.array_significant_figures
         dec_vals = ivy.array_decimal_values
-        if self.backend == "" or ivy.is_local():
```

Comment:
not sure why we're changing the `__repr__`, ig related to the same point about `numpy` being the implicit backend? This is something which could break if the array was belonging to a different backend than `numpy` right?